package/yast2-users.spec

```diff
@@ -39,7 +39,6 @@ BuildRequires:  yast2-core-devel
 BuildRequires:  yast2-devtools >= 4.2.2
 BuildRequires:  yast2-perl-bindings
 BuildRequires:  yast2-security
-BuildRequires:  yast2-testsuite
```

Comment:
I suspect perl-XML-Writer can also be removed now from the BuildRequires

Reply:
I agree. In fact, I was checking the dependencies right now.

Thanks for the _confirmation_ :)